Makefile

```diff
@@ -1,4 +1,10 @@
 APP_NAME=ether
+REGISTRY?=343660461351.dkr.ecr.us-east-2.amazonaws.com
+TAG?=latest
+ETHER_DB_LOCATION?=localhost:3306
+ETHER_DB_USERNAME?=ether
+ETHER_DB_PASSWORD?=ether
+ETHER_CONTENT_DIR=./
```

Comment:
Should this be `?=` or did you just want to declare the variable but not let it be changed?

Reply:
Should be `?=`. Just a mistake.